### ISSUE TITLE
.NET 6 support is missing.

### ISSUE THREAD

**Issue by bartsipes** [MicrosoftDocs/visualstudio-docs#8352]
The chart does not have .NET 6 listed despite the main page https://visualstudio.microsoft.com/vs/mac/ calling out support for .NET 6

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff562e19-8882-cfcf-7d92-85417535634f
* Version Independent ID: ff562e19-8882-cfcf-7d92-85417535634f
* Content: [.NET Core support - Visual Studio for Mac](https://docs.microsoft.com/en-us/visualstudio/mac/supported-versions-net?view=vsmac-2022#feedback)
* Content Source: [mac/supported-versions-net.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/mac/supported-versions-net.md)
* Product: **visual-studio-mac**
* Technology: **vs-ide-general**
* GitHub Login: @jmatthiesen
* Microsoft Alias: **jomatthi**

**Comment by AwyNaibaho72** [comment]
![20220609_085653](https://user-images.githubusercontent.com/88448130/184497001-e18cb980-6f88-4cd9-acea-bf14c0b91213.jpg)

**Comment by bartsipes** [comment]
Yes, this feedback is still actionable. Please also add .NET 7 now to the list as that has been released and also appears to be supported by Visual Studio Mac.

**Comment by bartsipes** [comment]
Yes, it's still valid. Just like I said the last time.

**Comment by j-martens** [comment]
#reassign: @anandmeg

**Comment by anandmeg** [comment]
@bartsipes, thanks for the feedback! We've updated the page and the changes should be live later today.

Thanks!
